resources/translations/en.edn

```diff
@@ -124,7 +124,7 @@
                           :returned "Changes requested"
                           :submitted "Applied"
                           :unknown "Unknown"}
-                 :intro [:div.intro [:p "By filling and sending in this application, all the applicants may be granted the applied resources. All the applicable licenses must also be accepted before or after applying."]]
+                 :intro [:div.intro [:p "By filling and sending in this application, the applicants may be granted the applied resources. All the applicable terms of use must also be accepted before or after applying by all the applicants."]]
```

Comment:
All the applicable terms of use must also be accepted before or after applying by all the applicants > All the applicable terms of use must also be accepted by each applicant before or after applying.

(A small change to the word order and a reduction in tautophony.)

Reply:
Made it even shorter.

Reply:
:+1: